### PR TITLE
Organization index and show

### PR DESCRIPTION
**Pull request by martinemde**
Index is a section of the dashboard.
Show is the organization view page.

We may need to separate out the private and public view of an org, but this is a rough start.

### CODE REVIEW

**Review by colby-swandale — COMMENTED** [review]
Everything looks good, but the CodeCov is picking up that we're missing a fair amount of tests for the various paths being added.

File: app/views/subscriptions/index.html.erb [hunk 1]
```diff
@@ -17,22 +17,26 @@
       <i><%= t("dashboards.show.no_subscriptions_html", :gem_link => link_to(t('dashboards.show.gem_link_text'), rubygem_path("rake"))) %></i>
     <% end %>
   <% else %>
-    <%= c.with_list(@subscribed_gems) do |gem| %>
-      <div class="flex flex-row w-full items-center justify-between">
-        <%= link_to rubygem_path(gem.slug) do %>
-          <h3 class="text-b1"><%= gem.name %></h3>
-          <p class="text-b3"><%= short_info(gem.most_recent_version) %></p>
+    <%= c.list do %>
+      <% @subscribed_gems.each do do |gem| %>
```
Comment (martinemde) [inline app/views/subscriptions/index.html.erb:21]:
```suggestion
      <% @subscribed_gems.each do |gem| %>
```

**Review by martinemde — COMMENTED** [review]
Syntax error

File: test/integration/organizations_test.rb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  setup do
+    @user = create(:user, remember_token_expires_at: Gemcutter::REMEMBER_FOR.from_now)
+    post session_path(session: { who: @user.handle, password: PasswordHelpers::SECURE_TEST_PASSWORD })
+  end
+
+  test "should show an organization" do
+    organization = create(:organization, owners: [@user], handle: "arrakis", name: "Arrakis")
+    organization.rubygems << create(:rubygem, name: "arrakis", number: "1.0.0")
+
+    get "/organizations/#{organization.to_param}"
```
Comment (colby-swandale) [inline test/integration/organizations_test.rb:13]: Can use we use the rails `organizations_path(organization)`, including below and other test files in this PR.

File: test/integration/organizations_test.rb [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+    assert_response :success
+    assert page.has_content? "arrakis"
```
Comment (colby-swandale) [inline test/integration/organizations_test.rb:16]: These assertions that are checking the Organization name/handle is a bit unclear in terms of what's specifically looking for.

```suggestion
    assert_selector "h1", @organization.title
```

File: test/integration/organizations/gems_test.rb [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    post session_path(session: { who: @user.handle, password: PasswordHelpers::SECURE_TEST_PASSWORD })
+  end
+
+  test "should get index" do
+    @organization = create(:organization, owners: [@user])
+    @organization.rubygems << create(:rubygem, name: "arrakis", number: "1.0.0")
+
+    get "/organizations/#{@organization.to_param}/gems"
+
+    assert page.has_content? "arrakis"
```
Comment (colby-swandale) [inline test/integration/organizations/gems_test.rb:15]:
```suggestion
   assert_content "arrakis"
```

**Review by colby-swandale — APPROVED** [review]
Everything looks good, just a few minor nitpick things

File: test/integration/organizations_test.rb (hunk 3 above)
Comment (martinemde) [reply test/integration/organizations_test.rb:16]: Can't make this change in this type of test :( Looking forward to your refactor.

File: test/integration/organizations_test.rb (hunk 2 above)
Comment (martinemde) [reply test/integration/organizations_test.rb:13]: Done, moved to `test/functional` which more correctly fits this pattern.

File: test/integration/organizations/gems_test.rb (hunk 4 above)
Comment (martinemde) [reply test/integration/organizations/gems_test.rb:15]: I think that works in other testing contexts and not here :( I hope we can resolve this discrepancy soon.

**Pull request merged**